templating/embedding_controllers.rst

```diff
@@ -98,4 +98,36 @@ string syntax for controllers (i.e. **bundle**:**controller**:**action**):
             ) ?>
         </div>
 
+If your controller should be used :doc:`as a service </controller/service>`,
+you can reference it like this instead:
```

Comment:
Here it goes. I was trying to embed a controller action in a twig template, this way:

    {{ render(controller('AppBundle:Default:completeSignup')) }}

Here is my  controller action:

    public function completeSignupAction(CategoryRepository $categoryRepository)
    {
        // ...
    }

And the CategoryRepository service definition

    services:
        _defaults:
            autowire: true
            autoconfigure: true
            public: false
    
        Domain\Model\CategoryRepository:
            factory: ['@doctrine.orm.entity_manager', 'getRepository']
            arguments: ['Domain\Model\Category']    

When I tried to access the page, I got the following error:

> An exception has been thrown during the rendering of a template ("Controller "Infrastructure\UI\Symfony\AppBundle\Controller\DefaultController::completeSignupAction()" requires that you provide a value for the "$categoryRepository" argument. Either the argument is nullable and no null value has been provided, no default value has been provided or because there is a non optional argument after this one.").

I then thought that maybe injecting a service in a controller action was not supported for subrequests (as I think that's what `render(controller())` does). I searched the docs but couldn't find anything about this, so I asked [on the support Slack channel](https://symfony-devs.slack.com/archives/C3EQ7S3MJ/p1497366746643674). This is where @greg0ire and others told me to try using the FQCN notation:

    {{ render(controller('Infrastructure\\UI\\Symfony\\AppBundle\\Controller\\DefaultController:completeSignupAction')) }}

This worked and my `$categoryRepository` was correctly injected, so @greg0ire created this PR.

I think this sums it all up.